view.py

```diff
@@ -0,0 +1,18 @@
+
+#view applicants
+class View: 
+
+    def __init__(self):
+        return
+    
+    def view_role(self,role):
+        print(f"The role is:{role}")
+
+    def view_applicants(self,applicants):
+        print(f"Here are all of the applicants:{applicants}")
```

Comment:
This prints the applicant list as one long list. Can you pretty print the applicant list like this:

```
Here are all of the applicants:

1. Emma Jones
2. Michael Bright
3. etc...
```